<launch>
    <!-- For Simulation -->
    <arg name="sim_mode" default="false" />
    <param name="/use_sim_time" value="$(arg sim_mode)"/>
    <arg name="loopTimes"       default="0" />
    <!-- move base -->
    <node pkg="robot_navigation" type="way_point.py" respawn="false" name="way_point" output="screen">
        <!-- params for move_base -->
        <param name="goalListX" value="[1.0, 1.0, 0.5, 0.5]" />
        <param name="goalListY" value="[0.0, 0.5, 0.5, 0.0]" />
        <param name="goalListZ" value="[0.707, 1.0, -0.707, 0.0]" />
        <param name="loopTimes" value="$(arg loopTimes)"/>
        <param name="map_frame" value="map" />
    </node>
    

</launch>
